queue_job/migrations/13.0.1.0.0/post-migration.py

```diff
@@ -0,0 +1,10 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade  # pylint: disable=W7936
+
+
+@openupgrade.migrate()
+def migrate(env, _version):
+    openupgrade.load_data(
+        env.cr, "queue_job", "migrations/13.0.1.0.0/noupdate_changes.xml"
+    )
```

Comment:
```suggestion
from odoo import SUPERUSER_ID, api


def migrate(cr, version):
    env = api.Environment(cr, SUPERUSER_ID, {})
    rule = env.ref("queue_job.queue_job_comp_rule", raise_if_not_found=False)
    if rule:
        rule.write({
            "domain_force": "['|',('company_id','=',False),('company_id','in',company_ids)]",
        })
```

Reply:
@astirpe there you go. Rename the folder to 13.0.1.0.1 and drop the additional file